core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -526,38 +395,42 @@ class WorkspaceService(
         }
       }
     )
-
-  private def maybeLoadMcWorkspace(workspaceContext: Workspace): Future[Option[WorkspaceDescription]] =
-    workspaceContext.workspaceType match {
-      case WorkspaceType.McWorkspace =>
-        Future(Option(workspaceManagerDAO.getWorkspace(workspaceContext.workspaceIdAsUUID, ctx)))
-      case WorkspaceType.RawlsWorkspace => Future(None)
-    }
+  }
 
   private def gatherWorkflowsToAbortAndSetStatusToAborted(workspaceName: WorkspaceName, workspaceContext: Workspace) =
-    dataSource.inTransaction { dataAccess =>
```

Comment:
I think this was just auto-formatted. I probably removed some unnecessary parenthesis or something, after re-enabling inspections on the file, causing the line breaks to change, or something like that.